modules/programs/direnv.nix

```diff
@@ -92,11 +93,9 @@ in {
     };
 
     xdg.configFile."direnv/direnvrc" = let
-      package =
-        pkgs.nix-direnv.override { inherit (cfg.nix-direnv) enableFlakes; };
```

Comment:
``` nix
if (functionArgs nix-direnv.override) ? enableFlakes then
  pkgs.nix-direnv.override { enableFlakes = true; }
else
  pkgs.nix-direnv;
```

Temporary "forward" compatibility?

Reply:
There's no need for that, `nixStable` now is `nix_2_4` which has flakes support, even without the linked PR.

The linked PR just gets rid of this option altogether

Reply:
:+1: